packages/core/src/lib/Upload/types.ts

```diff
@@ -9,8 +11,22 @@ export interface IUploadProps {
     /**
      * The callback function to be called onChange
      */
-
     onChange?: (file: Blob | undefined | null) => void;
+
+    /**
+     * Accepted file types (default: all)
+     */
+    acceptedFiles?: string;
```

Comment:
Could you add this link here in the comments so other devs can know what type of values can be used in this prop - https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept#:~:text=The%20accept%20attribute%20takes%20as,removed%20in%20favor%20of%20file.